SECURITY.md

```diff
@@ -16,3 +16,39 @@ As these 3rd party audits are completed and issues are sufficiently addressed, w
     - **Scopes**: *Ethereum Contracts, Solana Contracts, Terra Contracts, Guardian, and Solitaire*
 - **[July 1, 2022 - Kudelski](https://storage.googleapis.com/wormhole-audits/2022-07-01_kudelski.pdf)**
     - **Scopes**: *Ethereum Contracts, Solana Contracts, Terra Contracts, and Guardian*
+
+## White-Hat Hacking on Wormhole
+
+We want to lower the bar for White-hat hackers to find security bugs in Wormhole.  Why? The easier we make this process, the more likely it will be for white-hats to find bugs in Wormhole and responsibly disclose them, helping to secure the network.
+
+Here's a list of strategies that we've found helpful to hackers getting started on Wormhole:
+
+- Review the existing unit and integration testing (found in [CONTRIBUTING.md](https://github.com/certusone/wormhole/blob/dev.v2/CONTRIBUTING.md)) and see what we're already testing for.
+    * Check out places were we might be missing test coverage entirely.  This could be a ripe spot to look for something we missed.
+    * Check out places were we have unit/integration tests, but we lack sufficient [negative test](https://en.wikipedia.org/wiki/Negative_testing) coverage.
+- Review our different smart contract implementations (eg. Solana, EVM, CosmWasm) and attempt to understand how and why they are different.
```

Comment:
```suggestion
- Review our different smart contract implementations (eg. Solana, EVM, CosmWasm, Move) and attempt to understand how and why they are different.
```

Just a suggestion to add Move since there is already an existing Aptos PR.